store/sqlstore/upgrade.go

```diff
@@ -803,3 +808,81 @@ func upgradeDatabaseToVersion524(sqlStore SqlStore) {
 		saveSchemaVersion(sqlStore, VERSION_5_24_0)
 	}
 }
+
+func upgradeDatabaseToVersion525(sqlStore SqlStore) {
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_24_0, VERSION_5_25_0) {
+	saveSchemaVersion(sqlStore, VERSION_5_25_0)
+	//}
+}
+
+func upgradeDatabaseToVersion526(sqlStore SqlStore) {
+	//if shouldPerformUpgrade(sqlStore, VERSION_5_25_0, VERSION_5_26_0) {
+	err := precheckMigrationToVersion526(sqlStore)
+	if err != nil {
+		mlog.Error("Error upgrading DB schema to 5.26.0", mlog.Err(err))
+		os.Exit(EXIT_GENERIC_FAILURE)
+	}
+
+	sqlStore.AlterColumnTypeIfExists("Teams", "Type", "VARCHAR(255)", "VARCHAR(255)")
+	sqlStore.AlterColumnTypeIfExists("Teams", "SchemeId", "VARCHAR(26)", "VARCHAR(26)")
+	sqlStore.AlterColumnTypeIfExists("IncomingWebhooks", "Username", "varchar(255)", "varchar(255)")
+	sqlStore.AlterColumnTypeIfExists("IncomingWebhooks", "IconURL", "text", "varchar(1024)")
+
+	saveSchemaVersion(sqlStore, VERSION_5_26_0)
+	//}
+}
+
+func precheckMigrationToVersion526(sqlStore SqlStore) error {
+	teamsQuery, _, err := sqlStore.getQueryBuilder().Select(`CASE
```

Comment:
This could be done by the `AlterColumnTypeIfExists` but that would make some changes in the function that we should not tackle here.